parity-util-mem/derive/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "parity-util-mem-derive"
+version = "0.1.0"
+authors = ["Parity Technologies"]
+license = "GPL-3.0"
```

Comment:
[`malloc_size_of_derive`](https://github.com/bholley/malloc_size_of_derive) is MIT/Apach2 – if this is a clean re-implementation I guess it's fine to use GPL but, yeah, I think we should stick to MIT/Apache here.